pages/docs/open-source-roadmap/what-next.js

```diff
@@ -17,7 +17,7 @@ export default function DocsNext() {
         <h1 className="text-4xl mb-4 font-bold">
           Open Source Roadmap: What next?
         </h1>
-        <p>Coming soon</p>
+<p>It doesn&apos;t stop there! You can become a maintainer. If you want to do this for the communities you're in, you can do things like identifying duplicate issues or spam pull requests, aiding contributors with their pull requests, giving contributors friendly reminders about the Contributing Guide for first-time contributors. You can even take what you have learned and start your own Open Source project. The possibilities are endless.</p>
```

Comment:
My original suggestion isn't liked by React either sorry it should be `doesn\u0027t`